packages/ssl/ssl.0.7.0/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for OpenSSL"
+maintainer: ["Antonio Monteiro <anmonteiro@gmail.com>"]
+authors: ["Samuel Mimram <samuel.mimram@ens-lyon.org>"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/savonet/ocaml-ssl"
+bug-reports: "https://github.com/savonet/ocaml-ssl/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.03.0"}
+  "ocaml" {>= "4.08.0" & with-test}
+  "ocaml" {with_test & >= "4.08.0"}
```

Comment:
```suggestion
  "ocaml" {with_test & >= "4.08.0"}
```

Reply:
@mseri Merging this pretty sure there must have been some confusion.

Reply:
You are completely right, I had misread the file. It is not doing its job here though. I think we need a different solution

Reply:
> You are completely right, I had misread the file. It is not doing its job here though. I think we need a different solution

That's odd, it was working here: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/f8015193a0d6517ac62b2ff44b6a7c3f2fd3e4c7